### ISSUE TITLE
Server returned unexpected status code - 502 

### ISSUE THREAD

**Issue by Braunson** [cloudinary/cloudinary_php#33]
Running `\Cloudinary\Uploader::add_tag` and after a 35 calls, it returned 502 from Cloudinary.

More specifically:

```php
Fatal error: Uncaught exception 'Cloudinary\Error' with message 'Server returned unexpected status code - 502 - ' in /var/www/vhosts/staging/lib/cloudinary/cloudinary_php/src/Uploader.php:296 Stack trace:
#0 /var/www/vhosts/staging/lib/cloudinary/cloudinary_php/src/Uploader.php(225): Cloudinary\Uploader::call_api('tags', Array, Array) 
#1 /var/www/vhosts/staging/lib/cloudinary/cloudinary_php/src/Uploader.php(203): Cloudinary\Uploader::call_tags_api('gallery_159', 'add', Array, Array) 
#2 /var/www/vhosts/staging/www/assets/php/cron/gallery_tag_migration.php(57): Cloudinary\Uploader::add_tag('gallery_159', Array) 
#3 {main} thrown in /var/www/vhosts/staging/lib/cloudinary/cloudinary_php/src/Uploader.php on line 296
```

I noticed this was resolved in two other language libraries:
- https://github.com/cloudinary/cloudinary_gem/issues/118
- https://github.com/cloudinary/cloudinary_npm/issues/13

**Comment by tocker** [comment]
This may have been a result of the [Amazon S3 outage](http://www.information-age.com/technology/cloud-and-virtualisation/123459971/amazon-investigating-major-cloud-outage-github-and-heroku-report-issues). Can you please retest?

**Comment by Braunson** [comment]
@tocker Just ran it again, got a littler further but receives the same error.

**Comment by tocker** [comment]
Fixed in version 1.1.4. Can you please retest?
